policykit/policyengine/models.py

```diff
@@ -815,6 +815,11 @@ def pass_action(self):
         proposal.save()
         action.send(self, verb='was passed')
 
+    def get_applicable_policies(self):
+        if self.app_name == 'metagov':
+            return PlatformPolicy.objects.all()
+        return PlatformPolicy.objects.filter(community=self.community)
```

Comment:
I need to rethink this actually, because I didn't consider cases where a single PolicyKit instance is serving multiple communities.